static/app/views/issueDetails/actions/index.tsx

```diff
@@ -427,6 +427,7 @@ export function Actions(props: Props) {
             />
           </Fragment>
         ))}
+      {hasStreamlinedUI && <NewIssueExperienceButton />}
```

Comment:
do we need this `hasStreamlinedUI` check if we also check for it in the Button?

Reply:
i think it makes sense

Reply:
nvm see what you're saying

Reply:
It's so that it doesn't appear twice for the non streamlined header. It just has to do with how this component is laid out since the other render of `<NewIssueExperienceButton />` requires `hasStreamlinedUI` to be false.

Reply:
this might not work but could we have the button be controlled by the feature flag ? if they have the flag, they see the button , if not they don't. we could keep the query param so people could look at a single issue with the new design if they wanted to. i feel like im getting slightly confused at all the different pieces here and centralizing it around that might help

Reply:
That makes sense, it is getting complicated. Lemme try to simplify it then.

Reply:
Changed it and tests, the button is now only controlled by the organization flag. The actual UI is controlled by user preferences and the query param.